app/index.js

```diff
@@ -464,6 +490,7 @@ function initiateFeedbackData() {
     // Incremement the vote log by one
     votelog[0]++;
     console.log(votelog[0]);
+    voteLogLabel.text = votelog;
```

Comment:
Can we only do this in development mode? I don't see the advantage of displaying this in production?